queries/getIndexDocumentation.gql

```diff
@@ -0,0 +1,28 @@
+query getIndexDocumentation{
```

Comment:
i guess i'm at a loss as to what the current difference is intended to be between the index and [section] pages.  if we go with my proposal to have a listing of all the documentation articles instead of what is currently implemented, then i see the point of having a different function here, but atm this file is exactly the same as the other query.